### DISCUSSION TITLE
colouring of stack trace output?

### DISCUSSION THREAD

**Discussion by Mrodent** [borntyping/python-colorlog#121] [Support] [answered]
Thanks, nice job.

Just suggest a minor improvement: currently if you go `exception(e)` (passing an exception), the stack trace is printed out with the default (white) font.

I've tweaked my `ColoredFormatter` subclass so that method `format` substitutes these lines in:

```
        if record.exc_info:
            if not record.exc_text:
                
                # tweak so exception stack trace is bright red
                # record.exc_text = self.formatException(record.exc_info)
                record.exc_text = colorama.Style.BRIGHT + colorama.Fore.RED + \
                    self.formatException(record.exc_info) + colorama.Style.RESET_ALL

```
... I haven't looked at your source code, but there's probably a more elegant and "configured" way to do it, i.e. using the same colour as for example "ERROR" level, maybe by overriding `formatException` ...

**Comment by borntyping** [comment, marked as answer]
There's not currently any way to do this in `colorlog`. Overriding `formatException` is probably the right way to go if you wanted to add this feature.
